src/views/Students/StudentComponents/AlumniServiceForm.jsx

```diff
@@ -230,6 +260,81 @@ const AlumniServiceForm = (props) => {
                       required={feeFieldsRequired}
                     />
                   </div>
+                  <div className="col-md-6 col-sm-12 mt-2">
+                    <Input
+                      name="program_mode"
+                      label="Program Mode"
+                      placeholder="Program Mode"
+                      control="lookup"
+                      icon="down"
+                      className="form-control"
+                      options={programOptions}
+                    />
+                  </div>
+                  <div className="col-md-6 col-sm-12 mt-2">
+                    <Input
+                      name="category"
+                      label="Category"
+                      placeholder="category"
+                      control="lookup"
+                      icon="down"
+                      className="form-control"
+                      options={categoryOptions}
+                      onChange={handleCategoryChange}
```

Comment:
I think you can directly use an inline function here instead `handleCategoryChange` only contains one line.
```suggestion
                      onChange={(e) => setSelectedCategory(e.value)}
```